src/main.rs

```diff
@@ -81,7 +81,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         &cargo_toml_path,
         &crates_versions,
         cmd.overwrite,
-        !cmd.check,
+        cmd.check,
```

Comment:
The parameter to `update_dependencies` now is named `only_check`, so we should directly pass the `cmd.check` bool to the function, rather than the inverse.